docker/vcf_combiner/usr/bin/combine_vcf.py

```diff
@@ -254,18 +252,19 @@ def run(self):
                 raise CombineVCFException("Failed to combining VCF files: %s" % (err))
 
     @staticmethod
-    def get_version():
-        return "%s %s" % (ProductName, Version)
+    def get_my_name():
+        return ProductName
 
 def test_quick_test():
```

Comment:
tests whether code runs and has no syntax errors after pylint. rename test_quick_test()